graph/registry_credential_test.go

```diff
@@ -33,7 +33,7 @@ func TestCreateCredentialFromString(t *testing.T) {
 		{`{"registry":"r","identity":"clientID", "armResourceId": "https://management.azure.com"}`, true, &RegistryCredential{
 			Registry:      "r",
 			Identity:      "clientID",
-			ArmResourceID: "https://management.azure.com",
+			ArmResource: "https://management.azure.com",
```

Comment:
Could we just use the azure SDK DefaultBaseURI definition - https://github.com/Azure/azure-sdk-for-go/blob/master/services/resources/mgmt/2018-05-01/resources/client.go#L29

Reply:
Strictly speaking, baseuri is different from resource registered in AAD. For most resources, they might be same but arm is a special case.
Here, the string is used for test and can be anything. It doesn't really matter.

Reply:
The property at this point is not self describing. Is it’s a URI or a resourceId or an endpoint etc.